src/models/PlayerRanking.ts

```diff
@@ -1,5 +1,12 @@
+import { Team } from './Team'
+
 export interface PlayerRanking {
-  readonly name: string
   readonly id?: number
+  readonly name: string
+  readonly country?: string
+  readonly teams?: Team[]
```

Comment:
Is it possible to not have a team?

Reply:
You are right. I was not able to spot players without a team through this page (minMapCount = 0 to list every possible player) https://www.hltv.org/stats/players?minMapCount=0 

```
document.querySelectorAll('.teamCol').forEach(v => {
  if (v.querySelector('a') === null) {
    console.log(v)
  }
})
```